dag/main.yml

```diff
@@ -571,8 +571,14 @@ steps:
     - snapshot://wb/2024-06-10/gender_statistics.feather
   data://garden/wb/2024-06-10/gender_statistics:
     - data://meadow/wb/2024-06-10/gender_statistics
+  data://garden/wb/2024-06-10/gender_statistics_country_counts:
+    - data://garden/wb/2024-06-10/gender_statistics
+    - data://garden/regions/2023-01-01/regions
+    - data://garden/demography/2023-03-31/population
```

Comment:
You can now use the newest version of the population dataset: `data://garden/demography/2024-07-15/population`